data/tabular/train_test_split.py

```diff
@@ -197,40 +214,86 @@ def cli(
     # for path in paths_to_data:
     #     if "flashpoint" in path:
     #         filtered_paths.append(path)
-    #     elif "freesolv" in path:
+    #     elif "BBBP" in path:
     #         filtered_paths.append(path)
     #     elif "peptide" in path:
     #         filtered_paths.append(path)
+    #     elif "bicerano_dataset" in path:
+    #         filtered_paths.append(path)
     # paths_to_data = filtered_paths
 
     REPRESENTATION_LIST = []
 
-    for path in tqdm(paths_to_data):
+    for path in paths_to_data:
         df = pd.read_csv(path)
-        if repr_col in df.columns:
+
+        if repr_col in df.columns and path not in TRACKED_DATASETS:
+            print("Processing", path.split("/")[0])
+            TRACKED_DATASETS.append(path)
             REPRESENTATION_LIST.extend(df[repr_col].to_list())
 
     REPR_DF = pd.DataFrame()
-    REPR_DF["SMILES"] = list(set(REPRESENTATION_LIST))
+    REPR_DF[repr_col] = list(set(REPRESENTATION_LIST))
```

Comment:
in general, there should not need to be a need to allocate a dataframe for this (which can cost more memory). But it should work for our needs.